function alertError(errMsg) {
	Ext.Msg.alert('出错', errMsg);
}


